### PR TITLE
Rename tutorial docker

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

main-tutorial-humble-tutorial-source -> main-humble-tutorial-source

### CODE REVIEW

File: .docker/tutorial-source/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # syntax = docker/dockerfile:1.3
 
-# ghcr.io/ros-planning/moveit2:main-tutorial-${ROS_DISTRO}
+# ghcr.io/ros-planning/moveit2:main-${ROS_DISTRO}-tutorial-source
```
Comment (MarqRazz) [inline .docker/tutorial-source/Dockerfile:3]: Before this change the name was `ghcr.io/ros-planning/moveit2:main-tutorial-humble-tutorial-source`

Reading this I would expect the new name to be `ghcr.io/ros-planning/moveit2:main-humble-tutorial-source-tutorial-source` or am I missing something?

Comment (tylerjw) [reply .docker/tutorial-source/Dockerfile:3]: this is just a comment about what it is, ROS_DISTRO just expands to the ros distro name

File: .github/workflows/tutorial_docker.yaml [hunk 2]
```diff
@@ -21,8 +21,8 @@ jobs:
       packages: write
       contents: read
     env:
-      GH_IMAGE: ghcr.io/ros-planning/moveit2:main-tutorial-${{ matrix.ROS_DISTRO }}-${{ github.job }}
-      DH_IMAGE: moveit/moveit2:main-tutorial-${{ matrix.ROS_DISTRO }}-${{ github.job }}
+      GH_IMAGE: ghcr.io/ros-planning/moveit2:main-${{ matrix.ROS_DISTRO }}-${{ github.job }}
+      DH_IMAGE: moveit/moveit2:main-${{ matrix.ROS_DISTRO }}-${{ github.job }}
```
Comment (tylerjw) [inline .github/workflows/tutorial_docker.yaml:25]: @MarqRazz here is the real technical change.

ROS_DISTRO expands to `humble`, `rolling`, or `iron`
github.job expands to `tutorial-source`

so a full name will in the form `main-humble-tutorial-source`

File: .docker/tutorial-source/Dockerfile (hunk 1 above)
Comment (MarqRazz) [reply .docker/tutorial-source/Dockerfile:3]: I know the code only appends the ROS_DISTRO but where does it append the extra `-tutorial-source` at the end?

**Review by MarqRazz — APPROVED** [review]

**Pull request merged**